packages/instantsearch-ui-components/src/components/chat/types.ts

```diff
@@ -22,6 +22,7 @@ export type ClientSideToolComponentProps = {
   message: ChatToolMessage;
   indexUiState: object;
   setIndexUiState: (state: object) => void;
+  onClose: () => void;
```

Comment:
Doesn't need to be done now, but I think sending the entire chat ref would be a very flexible alternative to this if it's possible and clean.